src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -24,6 +29,47 @@ export class MarkdownToolbarComponent {
 	public buttonImage = localize('buttonImage', "Image");
 	public buttonPreview = localize('buttonPreview', "Markdown preview toggle - off");
 
-	constructor() {
+	public get cellModel(): ICellModel {
+		return this._cellModel;
+	}
+
+	@Input() public set cellModel(value: ICellModel) {
+		this._cellModel = value;
```

Comment:
Why both having separate gettings/setters/backing value when you could just put make the property public? That should still work with the @Input decorator.